### ISSUE TITLE
Turn off all output so that command output can be piped

### ISSUE THREAD

**Issue by smackesey** [tox-dev/tox#2763]
## What's the problem this feature will solve?

I am trying to pipe the output from running one of my `tox` environments to another tool. I cannot find a way to suppress _all_ tox output. Even with `tox -qqqqq` I always get e.g. this:

```
  pyright: OK (0.09=setup[0.08]+cmd[0.00] seconds)
  congratulations :) (0.36 seconds)
```

## Describe the solution you'd like

Make minimum verbosity mode suppress _all_ tox logging output.

## Alternative Solutions

Yes, by piping and dropping the last few lines with `awk`, but this is brittle and shouldn't be necessary.

**Comment by gaborbernat** [comment]
Can you describe your use case, please? What is exactly you're trying to achieve and why?

**Comment by smackesey** [comment]
>Can you describe your use case

I am using `tox` to run [pyright](https://github.com/microsoft/pyright) in a specific environment. In our case, pyright's output needs to be heavily massaged to be useful, so I need to pipe it to other programs.

**Comment by gaborbernat** [comment]
Can you expand on this?

**Comment by smackesey** [comment]
Pyright produces JSON on stdout which needs to be parsed, when tox appends a summary to the end this chokes the JSON parser.

But in any case-- while I first encountered this issue for what I've described above, I would probably be requesting this as a feature anyway. It is a common need to post-process the output of CLI tools, and it often makes sense to run certain CLI tools under specific environmental constraints using a tool like `tox`-- IMO it should be easy to get the straight output of the tool.
